jdl2makeflow

```diff
@@ -58,7 +58,7 @@ def preprocess_var(s, sh_mode):
 
 def gen_runjob(output_file, jdl, dry_run):
   runjob = j2env.from_string("""#!/bin/bash -e
-type zip unzip zipinfo
+type zip unzip zipinfo timeout
```

Comment:
Add `readlink` because people on the Internet are unsure about its ubiquitous presence...